.github/workflows/code_coverage.yml

```diff
@@ -0,0 +1,66 @@
+name: Code Coverage
+on:
+  pull_request:
+    branches-ignore: [dependabot/**]
+jobs:
+  comment-coverage-difference:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v2 # TODO: does this node setup need to happen each time we switch branches?
```

Comment:
I don't think it's needed to run this several times (just this one should be fine).